.github/CONTRIBUTING.md

```diff
@@ -51,6 +51,32 @@ We are running [mdformat](https://github.com/executablebooks/mdformat) for forma
 $ mdformat . --number
 ```
 
+### Test Coverage - Line and Branch
+
+We are using [Simplecov](https://github.com/colszowka/simplecov) to track test coverage.
+
+It is included and reported when you run `bundle exec rake`.
+
+To view the coverage report, open the `coverage/index.html` file in your browser. Here is a quick generic command to find and open the file on macOS:
+
+```console
+$ open $(find . -type f -path "*/coverage/index.html" -print -quit)
```

Comment:
This simpler version has never failed me:
```suggestion
$ open coverage/index.html
```
Plus, if the file doesn’t exist (before first spec run) you get a usable error message.

Reply:
Fair enough, I aliased this to work across nested repos at some point and never looked back.

`alias cov=...`